src/providers/twitch/api/Helix.cpp

```diff
@@ -315,6 +315,46 @@ void Helix::getGameById(QString gameId,
         failureCallback);
 }
 
+void Helix::createClip(QString channelId,
+                       ResultCallback<HelixClip> successCallback,
+                       HelixFailureCallback failureCallback)
+{
+    QUrlQuery urlQuery;
+    urlQuery.addQueryItem("broadcaster_id", channelId);
+
+    this->makeRequest("clips", urlQuery)
+        .type(NetworkRequestType::Post)
+        .header("Content-Type", "application/json")
+        .onSuccess([successCallback, failureCallback](auto result) -> Outcome {
+            auto root = result.parseJson();
+            auto data = root.value("data");
+
+            if (!data.isArray())
+            {
+                failureCallback();
+                return Failure;
+            }
+
+            std::vector<HelixClip> clips;
+
+            for (const auto &jsonClip : data.toArray())
+            {
+                clips.emplace_back(jsonClip.toObject());
+            }
+
+            successCallback(clips[0]);
+            return Success;
+        })
+        .onError([failureCallback](NetworkResult result) {
+            // clip creation failed LUL
```

Comment:
Pointless comment imo, since the code below it repeats the same information

Reply:
Yes I agree, I forgot to remove it after debugging, good catch :+1: